src/libs/BigNumberUtils.ts

```diff
@@ -16,4 +16,10 @@ export class BigNumberUtils {
   public static divideToGetWei(bn: BigNumber | string, number: string, units: number): BigNumber {
     return BigNumber.from(bn).div(parseUnits(number, units));
   }
+
+  public static convertTokenAmount(eth: string, sourceDecimals: number, targetDecimals: number) {
```

Comment:
How about changing `eth -> ethAmount`? It'll be clearer that it's an ETH amount. Plus, the second argument is named `sourceDecimals`, but we use `destToken.decimals` – that seems a bit off. Can we make it clearer what value needs to go in there?